js/core/dom_component.d.ts

```diff
@@ -8,14 +8,33 @@ import {
     DxElement,
 } from './element';
 
+import {
+    ChangedOptionInfo,
+    EventInfo,
+    InitializedEventInfo,
+} from '../events/index';
+
 import { TemplateManager } from './template_manager';
 import { FunctionTemplate } from './templates/function_template';
 import { DefaultOptionsRule } from './options';
 
+/** @public */
+export type DisposingEvent<TComponent> = EventInfo<TComponent>;
+
+/** @public */
+export type InitializedEvent<TComponent> = InitializedEventInfo<TComponent>;
+
+/** @public */
+export type OptionChangedEvent<TComponent> = EventInfo<TComponent> & ChangedOptionInfo;
+
```

Comment:
These types are not public - `dom_component` is a private module. Also, there is no need to export simple aliases

```suggestion

export type OptionChangedEventInfo<TComponent> = EventInfo<TComponent> & ChangedOptionInfo;

```